_data/mentors.yml

```diff
@@ -194,4 +194,13 @@
   github: hossainalikram
   image: hossain.jpg
   twitter: Hossainalikram
+  bio: 
+  
+  - name: Doris Lee
```

Comment:
@jig08 @roonyh I agree. No point reverting the PR now, there will be extra commits for reverting as well. We can fix it in future commits.